### PR TITLE
Fix GenericHandShader build errors

### PR DESCRIPTION
**Pull request by rblenkinsopp**
## Summary

Fixes a build error that was especially apparent when using "Build and Run" when compiling for Android.

## Contributor Tasks

_These tasks are for the merge request creator to tick off when creating a merge request._

- [x] Pair review with a member of the QA team.
- [ ] Add any release testing considerations to the MR for the next release.
- [x] Check any relevant CHANGELOG files have been updated.
- [x] Ensure documentation requirements are met e.g., public API is commented.
- [x] Consider any licensing/other legal implications for this MR e.g. notices required by any new libraries.
- [x] Add any relevant labels such as `breaking` to this MR.
- [x] If this MR closes a Jira issue, make sure the fix version on the JIRA issue is set to the correct one.

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

[Use emojis in review threads to communicate intent and help contributors.](https://github.com/ultraleap/UnityPlugin/blob/develop/CONTRIBUTING.md#review-threads)

- [x] Code reviewed.
- [x] Non-code assets e.g. Unity assets/scenes reviewed.
- [x] Documentation has been reviewed. Includes checking documentation requirements are met and not missing e.g., public API is commented.
- [ ] Checked and agree with release testing considerations added to MR for the next release.

## Closes JIRA Issue

- [UNITY-878](https://ultrahaptics.atlassian.net/browse/UNITY-878)

### CODE REVIEW

**Review by MattGrayUL — APPROVED** [review]
Tested this while building and also tested the ghost and outline hands in Unity 2020 to check for regressions, all looks fine :)

**Comment by MattGrayUL** [comment]
@Greg-UL are you happy with the coverage of my testing here? This shader is default rendering only so the only thing not covered here is Unity 2021, but it would be essentially testing Unitys code at that point

File: Packages/Tracking/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [docs-website]: https://docs.ultraleap.com/unity-api/ "Ultraleap Docs"
 
+## [6.2.0]
```
Comment (MattGrayUL) [inline Packages/Tracking/CHANGELOG.md:9]: This should be `[NEXT] - Unreleased` just in case :)

**Comment by Greg-UL** [comment]
Can we just retest cross platform on this one please given the nature of this being more prominent on Android?

**Comment by Greg-UL** [comment]
Okay, then in that case i'm happy with that.

**Pull request merged**